src/components/Sections/VideoSection/__tests__/VideoSection.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { render } from "@testing-library/vue";
+import VideoSection from "..";
+
+const dummyId = "N0tRe4lYTID";
+const testYtUrl = "http://not-a-real-url.org/player/";
+
+describe("components/sections/video-section", () => {
```

Comment:
We should include a test to make sure that the VideoSection can handle a youtubeUrl without a slash at the end.

Reply:
Done.